src/views/404.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="container">
+    <br><br>
+    <div class="row">
+      <div class="col-md-4">
+        <img class="wfcd-logo"  name="WFCD Logo" :src="wfcdLogo" />
+      </div>
+
+      <div class="col-md-8">
+        <h1>404 ERROR OH NO</h1>
+        <h2>You broke the site!</h2>
+        <h3>How could you?</h3>
+
+        <br><br>
+        <p>Don't worry, one of the developer will clean up the mess, you can carry on though!</p>
```

Comment:
For better grammar: "Don't worry, one of the developer**s** will clean up the mess**. Y**ou can carry on though!"

Reply:
Sure, sounds good.